gamepad_wrapper.py

```diff
@@ -1,11 +1,18 @@
-from evdev import InputDevice, categorize, ecodes
+from evdev import InputDevice, categorize, ecodes, list_devices
 import psutil
 from subprocess import Popen
 
 #creates object 'gamepad' to store the data
 #you can call it whatever you like
 # you will need to check /dev/input for your specific device name like eventXX
-gamepad = InputDevice('/dev/input/event15')
+
+
+devices = [InputDevice(path) for path in list_devices()]
+for device in devices:
+   if device.name == "Microntek              USB Joystick          ":
```

Comment:
Change the device name here to your controller name

Reply:
So I change device name to 8BitDO SN30 Pro?

What if I have different controllers? Am I able to add additional fields there?